api/SpeechSynthesisEvent.json

```diff
@@ -238,10 +238,62 @@
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "milliseconds": {
+          "__compat": {
+            "description": "<code>elapsedTime</code> in milliseconds",
+            "support": {
+              "chrome": {
+                "version_added": "33"
+              },
+              "chrome_android": {
+                "version_added": "33"
+              },
+              "edge": {
+                "version_added": "14"
+              },
+              "firefox": {
+                "version_added": "49",
+                "version_removed": "95"
+              },
+              "firefox_android": {
+                "version_added": "62",
+                "version_removed": "95"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": "21"
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": "7",
+                "version_removed": "14.1"
+              },
+              "safari_ios": {
+                "version_added": "7",
+                "version_removed": "14.5"
```

Comment:
iOS version matching desktop 14.1

Reply:
Firefox for Linux and Firefox for Android is that `elapsedTime` is second even if old version.  [Bug 1732498](https://bugzilla.mozilla.org/show_bug.cgi?id=1732498) is Windows and macOS only.